### PR TITLE
feat: Add Network Layer for making api calls.

### PR DESCRIPTION
**Pull request by Gautam-Arora24**
Introduced ESlint config in the project to have consistent linting.

Fixes: #62

### CODE REVIEW

File: src/views/Home.vue [hunk 1]
```diff
@@ -6,7 +6,10 @@
 
 <script>
 export default {
-  name: 'Home',
+  name : "Home",
+  mounted() {
+    console.log( process.env.VUE_APP_fuck );
```
Comment (SBagaria2710) [inline src/views/Home.vue:11]: A little NSFW, don't you think?

Comment (Gautam-Arora24) [reply src/views/Home.vue:11]: LoL, was just testing env variables. Just making a followup commit 😅

**Review by SBagaria2710 — APPROVED** [review]

File: src/api/index.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+const httpClient = axios.create( {
+  baseURL : process.env.BASE_URL,
```
Comment (samarpan1738) [inline src/api/index.js:4]: We can move the base url in constants file and import it from there

Comment (Gautam-Arora24) [reply src/api/index.js:4]: Hey @samarpan1738! So, these environment variables can be added inside an `.env` file with `VUE_APP` as a prefix to each key, and then can be later used in the whole app. 

So, answering to your question, the base URL has already been added to the env file and therefore we are able to access it using `process.env`. 
@SBagaria2710 Can we add an example_env file in the project to make it clear for new devs?

**Pull request merged**